src/olympia/scanners/models.py

```diff
@@ -99,6 +99,11 @@ def can_report_feedback(self):
             self.has_matches and self.state == UNKNOWN and self.scanner != WAT
         )
 
+    def can_revert_feedback(self):
```

Comment:
We should probably have one or two test cases for this method.

Reply:
What exactly should be tested?

Reply:
- a scanner result that has been previously reported as false or true positive can be reverted
- a default scanner result should not be "revert-able"

Reply:
4b1f07febe